packages/types/src/json/Header.001.json

```diff
@@ -0,0 +1 @@
+{"jsonrpc":"2.0","result":{"digest":{"logs":[]},"extrinsicsRoot":"0x45b0cfc220ceec5b7c1c62c4d4193d38e4eba48e8815729ce75f9c0ab0e4c1c0","number":0,"parentHash":"0x0000000000000000000000000000000000000000000000000000000000000000","stateRoot":"0x294c3470ae3be7555240b9d034ec19c3715ba2c3f20b92441f8cea0cab66ab56"},"id":1}
```

Comment:
This one was edited a bit, i.e. turned it into a valid rpc json reponse, hence not being a straight rename like the others.